ops/github_telemetry/discussions.py

```
@@ -0,0 +1,108 @@
+    if r.returncode != 0:
+        print(f"discussions collect failed: {r.stderr[:500]}", file=sys.stderr)
+        return 2
+    try:
+        raw = json.loads(r.stdout)
+    except json.JSONDecodeError as e:
+        print(f"discussions JSON error: {e}", file=sys.stderr)
+        return 2
+    out.mkdir(parents=True, exist_ok=True)
+    raw_path = out / f"discussions-raw-{stamp}.json"
+    raw_path.write_text(
+        json.dumps({"collected_at": datetime.now(timezone.utc).isoformat(), "data": raw}, indent=2) + "\n"
```

Pylint (4.0.7)

[convention] Line too long (106/100)

(C0301)